testing/wireguard-grsec/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+
+_flavor=${FLAVOR:-grsec}
+_kpkg=linux-$_flavor
+_kver=4.9.13
+_kpkgrel=0
+
+# when changing _ver we *must* bump _mypkgrel
+_ver=0.0.20170223
+_mypkgrel=0
+_name=wireguard
+
+# verify the kernel version before entering chroot
+_kapkbuild=../../linux-${_flavor}/APKBUILD
+if [ -f $_kapkbuild ]; then
+	. $_kapkbuild
+	pkgname=$_name-$_flavor
+	[ "$_kver" != "$pkgver" ] && die "please update _kver to $pkgver"
+	[ "$_kpkgrel" != "$pkgrel" ] && die "please update _kpkgrel to $pkgrel"
+fi
+
+_kpkgver="$_kver-r$_kpkgrel"
+_abi_release=${_kver}-${_kpkgrel}-${_flavor}
+
+pkgname=${_name}-${_flavor}
+pkgver=$_kver
+pkgrel=$(($_kpkgrel + $_mypkgrel))
+pkgdesc="Next generation secure network tunnel: kernel modules for $_flavor"
+arch='all'
+url='https://www.wireguard.io'
+license="GPLv2"
+makedepends="linux-grsec-dev=$_kpkgver libmnl-dev sparse"
+source="https://git.zx2c4.com/WireGuard/snapshot/WireGuard-$_ver.tar.xz"
+builddir="$srcdir"/WireGuard-$_ver
+
+build() {
+	cd "$builddir"
+	# only building module: see wireguard-tools for userspace
+	make -C src/ \
+		KERNELDIR=/lib/modules/${_abi_release}/build \
+		module \
+		|| return 1
+}
+
+package() {
+	cd "$builddir/src"
+	install_if="linux-${_flavor}=${_kver} $pkgname"
+
+	local module=
+	for module in *.ko; do
+		install -v -D -m644 ${module} \
+			"$pkgdir/lib/modules/$_abi_release/extra/${module}"
+	done
+}
+
+check() {
+	return 0
+	# currently failing: attribute 'nocapture': unknown attribute
+	# not part of musl libc ?
+	make -C src/ \
```

Comment:
Just remove this section. It doesn't do what you want in the first place. It's for development, not for usage by packagers. If this confusion becomes widespread, I'll just remove it from the package I distribute and keep the shortcut locally.